src/presentation/styles/framerMotionComponents/AnimationConstants.js

```diff
@@ -0,0 +1,183 @@
+import { motion } from "framer-motion"
+
+// Defining reference constants for animation of various components (to ensure style & motion consistency)
+export const easing = [0.6, -0.05, 0.01, 0.99];
+export const transition = { duration: 1, ease: [0.6, 0.01, -0.05, 0.9] };
+
+// Have multiple grouped animated elements appear at slightly offset times.
+export const stagger = {
+    animate: {
+        transition: {
+            delayChildren: 0.4,
+            staggerChildren: 0.2,
+            staggerDirection: 1,
+        }
+    }
+};
+
+// Defining reference constants for animation of various components (to ensure style & motion consistency)
```

Comment:
you could possibly be more specific here with why you chose some of these constant values and what the values themselves mean in terms of the animation